src/Api/ApiResponse.php

```diff
@@ -36,4 +52,13 @@ public function withError (?string $error) : self
 		$this->error = $error;
 		return $this;
 	}
+
+
+	/**
+	 *
+	 */
+	public function isOk () : bool
+	{
+		return $this->statusCode >= 200 && $this->statusCode <= 299;
```

Comment:
I guess we don't want to use the `ApiResponse` for something like a 304, this is fine.

Reply:
Using it for a `3xx` response makes no sense, as a redirect response needs a `Location` header and may not have a body.

Reply:
Even a REST API wouldn't use the payload / I would consider that a code smell.
What should it do with all the payloads it collects while following the redirects?